setup.sh

```diff
@@ -1,24 +1,242 @@
 #!/bin/sh
 
-if ! [ $(id -u) = 0 ]; then
-   	printf "Please run as root to proceed with installations... \n1. Type in: su\n2. Enter your password\n3. Execute: sh setup.sh\n"
+YUM=$(which yum)
+DNF=$(which dnf)
+APT_GET=$(which apt-get)
+PACMAN=$(which pacman)
+BREW=$(which brew)
 
```

Comment:
@kmehant it would be great if we can have files for each OS/distribution like we have right now for Ubuntu and Arch so that managing would be better and all the packages will listed at seperate place. @agentmilindu what do you say?

Reply:
Yeah, agree with @vivionk

Reply:
@vivonk @agentmilindu 
I have updated dependency installation for each operating system in to seperate executable files. Please review.